[2022-08-06 03:39:31,781] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-05T00:00:00+00:00 [queued]>
[2022-08-06 03:39:31,791] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-05T00:00:00+00:00 [queued]>
[2022-08-06 03:39:31,792] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 03:39:31,792] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-06 03:39:31,793] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 03:39:31,808] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): final_dag> on 2022-08-05 00:00:00+00:00
[2022-08-06 03:39:31,820] {standard_task_runner.py:52} INFO - Started process 9618 to run task
[2022-08-06 03:39:31,823] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'final_dag', 'scheduled__2022-08-05T00:00:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpooh98jj0', '--error-file', '/tmp/tmpxqgxnk9b']
[2022-08-06 03:39:31,824] {standard_task_runner.py:80} INFO - Job 135: Subtask final_dag
[2022-08-06 03:39:31,894] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.final_dag scheduled__2022-08-05T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-06 03:39:31,970] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=final_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-05T00:00:00+00:00
[2022-08-06 03:39:32,054] {python.py:173} INFO - Done. Returned value was: None
[2022-08-06 03:39:32,068] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=final_dag, execution_date=20220805T000000, start_date=20220806T033931, end_date=20220806T033932
[2022-08-06 03:39:32,116] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 03:39:32,148] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
